media/aplResourceSync/aplResourceSync.js

```diff
@@ -7,33 +7,34 @@
 /* eslint-disable no-undef */
 const vscode = acquireVsCodeApi();
 
-window.onload = function() {
+window.onload = function () {
     retrieve();
     window.addEventListener('message', event => {
         const message = event.data;
         if (message.names) {
             const selectList = document.getElementById('name');
-            selectList.innerHTML = "";
+            selectList.innerHTML = '';
             message.names.forEach(name => {
-                var option = document.createElement("option");
+                var option = document.createElement('option');
                 option.value = name;
                 option.text = name;
                 selectList.appendChild(option);
             });
         }
     });
 
-    document.getElementById("downloadApl").onsubmit = function downloadApl() { 
+    document.getElementById('downloadApl').onsubmit = function downloadApl() {
         const name = document.getElementById('name').value;
         vscode.postMessage({
             action: 'sync',
             name: name,
         });
+        return false;
```

Comment:
why return false ? this function should return void ? Is this a new requirement also ?

Reply:
No. Here, we are listening to the `submit` event and triggering an action. If we return void, it means the submission checks are fine and the form should be submitted to the next navigation step. However, that is not the case in these webviews since we perform navigation through backend.

This is not a new requirement from vscode, but instead seems to be related to the vscode behavior change when changed from the `webview` tag to `iframe` tag